src/App.svelte

```diff
@@ -0,0 +1,494 @@
+<script>
+// import { onMount } from "svelte";
+let value = 0;
+let counter = 1;
+let themeValue = 1;
+let storage = [0,0];
+let operator = false;
```

Comment:
What is the use of this variable? Naming is not clear enough.

Reply:
Changed with "anyOperationDone"